platform-coop-toolkit/coop-map-directory-index
# feat: add Category.category_group

## Description

Adds a field to the Category model which allows categories to be grouped into one of three slices:

- Cooperatives
- Potential cooperatives (e.g. businesses exploring the co-op model)
- Other organizations (e.g. academic institutions, consultancies, and other support organizations)

## Steps to test

1. Check out branch.
2. Run the migration.
3. Examine the Category model in the admin UI.

**Expected behavior:** Categories can be assigned to one of these three groups.

## Additional information

Not applicable.

## Related issues

- Blocks #84.
